src/js/views/settingsView.js

```diff
@@ -0,0 +1,124 @@
+var Marionette = require('backbone.marionette'),
+    _ = require('underscore'),
+    $ = require('jquery'),
+    bindAll = require('lodash.bindall');
+
+var template = require('../../templates/settings.tpl');
+
+module.exports = Marionette.ItemView.extend({
+    id: 'sk-settings',
+    template: template,
+
+    ui: {
+        email: '[data-ui-email]',
+        saveButton: '[data-ui-save]',
+        formMessage: '[data-ui-form-message]',
+        form: '[data-ui-form]'
+    },
+
+    behaviors: {
+        stickit: {
+            '@ui.email': {
+                observe: 'email',
+                onSet: 'onEmailSet',
+                update: 'emailUpdate'
+            }
+        }
+    },
+
+    triggers: {
+        'click @ui.saveButton': 'settings:save'
+    },
+
+    events: {
+        'submit @ui.form': 'onFormSubmit'
+    },
+
+    initialize: function() {
+        bindAll(this);
+    },
+
+    emailUpdate: function($el, val, model, options) {
+        $el.val(val);
+
+        if (this.getOption('readOnlyEmail')) {
+            $el.attr('disabled', true);
+        }
+    },
+
+    onEmailSet: function(value) {
+        // http://stackoverflow.com/questions/46155/validate-email-address-in-javascript
+        var regex = /^(([^<>()[\]\\.,;:\s@\"]+(\.[^<>()[\]\\.,;:\s@\"]+)*)|(\".+\"))@[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?(?:\.[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?)*$/;
+
+        var isValid = regex.test(value);
```

Comment:
I thought about it, but we need to keep the widget footprint as low as possible. It felt acceptable to go with that regex for now. If things are becoming more complex, I'm all for bringing in Backbone.Validation and the behavior we have in the app.